Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -3155,7 +3155,7 @@ jQuery(($) => {
                     {
                         title: 'Roles',
                         text: `Here you can see all the existing roles and edit their permissions. All users will be in
-                            the "Anonymous" role by default. Once they log in, they'll have the "Authenticated" role
+                            the Anonymous role by default. Once they log in, they'll have the Authenticated role
```

Comment:
Since these are referring to (role) names, as opposed to quoting a UI label (like in "once done, click on the "Save" button"), I don't think quotes are necessary (or useful). But `i` is, I'm adding it.